packages/@atjson/document/src/annotation.ts

```diff
@@ -60,7 +60,7 @@ export interface AnnotationConstructor<T, Attributes> {
   }): T;
 }
 
-export default abstract class Annotation<Attributes = {}> {
+export default abstract class Annotation<Attributes = { [key: string]: any }> {
```

Comment:
🙅 Let's keep this as-is. The solution for this is to use `Annotation<any>` when you want to use this generically.

Otherwise, this would require us to change our type definitions for things like `Bold`. There's a default implementation for annotations that expects an empty object `{}`.